thirdparty/fetch-thirdparty-deps-windows.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# Die on any error for Travis CI to automatically retry:
```

Comment:
Actually, the other way round. This should get merged first as https://github.com/OpenRA/OpenRA/pull/7345 has to adapt to the changes here.